source/mir/ndslice/slice.d

```diff
@@ -2778,6 +2790,18 @@ pure nothrow unittest
     }
 }
 
+unittest
+{
+    // test save with dynamic arrays
```

Comment:
dynamic arrays are handled differently in contrast to ranges, thus the extra test.